aggsender/flows/flow_aggchain_prover_test.go

```diff
@@ -120,7 +121,7 @@ func Test_AggchainProverFlow_GetCertificateBuildParams(t *testing.T) {
 					nil,
 				)
 				mockL1InfoDataQuery.On("CheckIfClaimsArePartOfFinalizedL1InfoTree", mock.Anything, mock.Anything).Return(nil)
-				mockChainGERReader.On("GetInjectedGERsForRange", ctx, uint64(1), uint64(10)).Return(map[uint64][]common.Hash{}, nil)
+				mockChainGERReader.On("GetInjectedGERsForRange", ctx, uint64(1), uint64(10)).Return(map[uint64][]chaingerreader.InjectedGER{}, nil)
```

Comment:
Nitpick, you could use expecter (`EXPECT()`) here, as it is more developer friendly.

Reply:
I agree, but I would leave that for some cleanup PR, so we can change all UTs to use expecter.

Reply:
Done: https://github.com/agglayer/aggkit/pull/377/commits/689daf2ad01a379704262ac491e30292843344d5